pkg/virt-launcher/virtwrap/network/common.go

```diff
@@ -196,11 +196,11 @@ func (h *NetworkUtilsHandler) ConfigureIpForwarding(proto iptables.Protocol) err
 }
 
 func (h *NetworkUtilsHandler) IsIpv6Enabled(interfaceName string) (bool, error) {
-	link, err := Handler.LinkByName(interfaceName)
+	link, err := h.LinkByName(interfaceName)
```

Comment:
The commit message confused me; could you indicate you're using the receiver ? The 'itself' word threw me off.

Reply:
Done